src/components/SingleCard.css

```diff
@@ -7,6 +7,10 @@
     border-radius: 6px;
 }
 
+.image{
+    background-size: contain;
```

Comment:
Seems like this break some pictures: 
| Your change  | Production |
| ------------- | ------------- |
| ![grafik](https://github.com/idna001/Couple-Memory/assets/46223359/6416d6bc-645b-4f21-8b6e-8c5a38f5b691)  | ![grafik](https://github.com/idna001/Couple-Memory/assets/46223359/54468281-9f76-43e8-946f-47cb10360c5c)  |